docs/source/releases/v1_13_0.rst

```diff
@@ -0,0 +1,264 @@
+.. dropdown:: Distribution Statement
+
+ | # # # Distribution Statement A. Approved for public release. Distribution is unlimited.
+ | # # #
+ | # # # Author:
+ | # # # Naval Research Laboratory, Marine Meteorology Division
+ | # # #
+ | # # # This program is free software: you can redistribute it and/or modify it under
+ | # # # the terms of the NRLMMD License included with this program. This program is
+ | # # # distributed WITHOUT ANY WARRANTY; without even the implied warranty of
+ | # # # MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the included license
+ | # # # for more details. If you did not receive the license, for more information see:
+ | # # # https://github.com/U-S-NRL-Marine-Meteorology-Division/
+
+Version 1.13.0 (2024-06-08)
+**************************************
+
+* Update internal 'update_this_release_note' to v1_13_1.rst
+* Update github.com 'update_this_release_note' to v1_14_0a0.rst
+* Bug fixes
+
+  * Add create_plugin_registries call to full_install.sh
+  * Remove additional references to get_entry_point_group
+  * Add himawari9 to test data downloads
+  * Update geokompsat unit test to use test_data_noaa_aws
+* Documentation Updates
+
+  * Ensure :orphan: tags are on the first line of rst files
+* Testing Updates
+
+  * Add ami and cli test scripts to full_test and test_all
+  * Update AMI TC test script to use $GEOIPS test bdeck file
+  * Update test outputs for cartopy 0.23.0 and matplotlib 3.9
+  * Remove all references to geokompsat_netcdf reader (replaced with ami_netcdf)
+  * Replace check-old-docs-frozen.py with git-based Actions check
```

Comment:
I didn't touch this test. I left it as-is since it works, is temporary, and has no need of the docker image. It's still in `check-old-docs-frozen.yaml`.